upstash/manifest.json

```
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b6204e82-0732-4ab4-bca6-ed2e727ea581",
+  "app_id": "upstash",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Publish your Upstash Redis metrics to Datadog",
+    "title": "Upstash",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upstash Database Metrics",
+        "image_url": "assets/images/upstash-dashboard.png"
```

The url to this image is invalid:
```suggestion
        "image_url": "images/upstash-dashboard.png"
```